### ISSUE TITLE
Tenor Api key not working

### ISSUE THREAD

**Issue by arjavpd** [Thanasis1101/react-native-gif-search#19]
Hi! I can't get my tenor api key to work with gif search. I got the key from my google console and it's still not working. I feel like I'm missing something big but idk what. I'm pretty stuck so any help is appreciated! 😭

**Comment by rogueloop** [comment]
Did it get fixed? @arjavpd

**Comment by arjavpd** [comment]
nope. got a api key from https://tenor.com/gifapi/documentation#quickstart and then pasted it in my gif search component but it didn't work :(

**Comment by rogueloop** [comment]
I think the issue is with tenor V2 tokens, here they use V1 API call for tenor. 
https://github.com/Thanasis1101/react-native-gif-search/blob/ccc124ff20ae55606f38cc66133c715e78c6f3e1/src/GifSearch.js#L19

**Comment by arjavpd** [comment]
looks like we're gonna have to patch the module, @rogueloop have you done that?

**Comment by rogueloop** [comment]
Created a new one

https://twitter.com/d4rkcyph3r/status/1655904564095356929?t=rJ3SKrPWFusF6yxUTHvkMA&s=19

**Comment by yadavmurari111** [comment]
I am also facing same problem with tenor V2 tokens, got the tenor api key from here : https://tenor.com/gifapi/documentation#quickstart but it is not working , please suggest me any possible solution ?

**Comment by rogueloop** [comment]
Try this plugin https://github.com/HexmosTech/expo-tenor-search
